website/docs/getting-started.md

```diff
@@ -0,0 +1,153 @@
+
+To send a message to a channel, for example, you can call the `bot.helpers.sendMessage` method with the 2 required parameters:
+
+- The channel ID corresponding to the channel where you want your message to be
+- The message options to create your message
+
+```ts
+// Change the id below with one of the channels of one of the server your bot is in
+const message = await bot.helpers.sendMessage(123123123123123123n, {
+  content: 'Hello world. This is test message from Discordeno.',
+})
+```
+
+:::tip[Naming of functions in Discordeno]
+Most of the functions in Discordeno tend to be very explicit in what they will do and be similar to how discord calls their endpoints in the documentation.
+
+Also helpers methods from Discordeno will always perform a single action. They will never call multiple Discord endpoints.
+:::
+
+## Understanding Desired Proprieties in Discordeno
+
+If we ran the code from above, for example on the ready event, the bot will send a message in the channel you specified, as long as it has the permissions to do so, with the content of "Hello world. This is test message from Discordeno.", however if we log to the console the message object we are getting we won't see many, if any, values on it. This is also the case from the events such as `MessageCreate`, but why is that? Well, this is a Discordeno feature called `Desired proprieties`.
+
+Desired proprieties are a feature to allow for a smaller memory impact on your application for proprieties that you don't use. An example may be the channel topic, you might not be interested in knowing the topic of a channel, however Discord will always give it to you making your code have to deal with values that you will not use but still consume memory. This is why Discordeno requires you to explicitly set ahead of time the proprieties that you want/use.
+
+You can set these from the `bot.transformers.desiredProprieties` object. This will contain an object for every type that supports this feature, such as `Message` or `Interaction` for example where inside you will find the single proprieties with a boolean value, or an object with more proprieties inside. Discordeno will default everything to false and you can set the values you need to true manually like this:
```

Comment:
```suggestion
You can set these from the `bot.transformers.desiredProperties` object. This will contain an object for every type that supports this feature, such as `Message` or `Interaction` for example, inside which you will find the single properties with a boolean value, or an object with more properties inside. Discordeno will default everything to false and you can set the values you need to true manually like this:
```